### DISCUSSION TITLE
Is there a way to optionally mask characters?

### DISCUSSION THREAD

**Discussion by brandonleichty** [beholdr/maska#167] [Q&A]
Hello 👋

I'm wondering if there's a way to optionally mask characters using Maska? 

For example, let's say we have a fake credit card number like this: `4012888888881881`. 
Is there a way to apply a mask with a character such as `•` so that the card number looks like: `••••••••••••1881`?

I'd like to ideally be able to toggle between `4012888888881881` and  `••••••••••••1881` being displayed (maybe some Maska attribute could be added/removed)?

Thank you! 🙏

**Comment by beholdr** [comment]
You can create custom token to allow `•` and digits. Like:

``` html
<input v-maska data-maska="################" data-maska-tokens="#:[•0-9]">
```

**Reply by brandonleichty** [reply]
What if I’m not looking to allow the user to type/enter `•`, but rather mask the characters they type with `•`?

**Reply by beholdr** [reply]
Then use `type="password"` :)

**Reply by brandonleichty** [reply]
Haha yeah, that would make sense if it was a password :) 

What I used above was just an example. So it doesn’t sound like there’s a way to mask a users input using other characters?

**Reply by beholdr** [reply]
I'm afraid not, for such complex masking you need to have different states for input/output characters.
